tests/test_joint_logprob.py

```diff
@@ -241,6 +243,7 @@ def test_warn_random_not_found():
         conditional_logprob(y_rv, warn_missing_rvs=False)
 
 
+@pytest.mark.skip(reason="Disabled until we decide how we want this to work.")
```

Comment:
I don't see why this one shouldn't work. There is no ambiguity there, the two logdensities can be bound to the same value.

Reply:
Thoughts?

Reply:
Yeah, I agree; I'll change/update this one.